BrainPortal/lib/cbrain_task_generators/schema_task_generator.rb

```diff
@@ -166,6 +166,20 @@ def integrate(register: true, create_tool_config: false, multi_version: false)
         })[partial]
       end
 
+      # Write out a help file for this Boutiques task
+      helpFileName = name + "_help.html" 
+      helpFileDir = "cbrain_plugins/cbrain_tasks/help_files/"
```

Comment:
There is a (tacit?) convention in CBRAIN that we try to align '=' signs as much as possible.